proxy/api/src/project.rs

```diff
@@ -105,6 +108,26 @@ pub fn list_projects(api: &coco::Api) -> Result<Vec<Project>, error::Error> {
         .collect()
 }
 
+/// List all projects you have contributed to.
+///
+/// # Errors
+///
+///   * We couldn't get a project list
+///   * We couldn't get project stats
+pub fn list_my_projects(api: &coco::Api) -> Result<Vec<Project>, error::Error> {
```

Comment:
As mentioned in the context of the http changes, could this be generalised to a method which, given a user, return the list of projects that user has contributed to (contribution here as per definition in the PR description).

Reply:
Thinking about this, and I swear I had this idea before but maybe I miscommunicated or left it out, but I feel like a lot of this could be cleared up by creating a variant of [`all_metadata`](https://github.com/radicle-dev/radicle-link/blob/master/librad/src/git/storage.rs#L231-L241), called `all_metadata_of` that filters the list based on `remotes/<peer_id>`. 

That way you pass a `PeerId` when you want to look at specific profile.

Reply:
That makes a lot of sense to me! 

> That way you pass a `PeerId` when you want to look at specific profile.

Wouldn't we need to multiple calls for all `PeerId`s we know for the user?

Reply:
radicle-dev/radicle-link#306

Reply:
> Wouldn't we need to multiple calls for all PeerIds we know for the user?

Potentially, but I think it might be safe to assume we're looking at one user for now. I think that simplification is assumed in a few places at this stage.